src/test/regress/sql/multi_cluster_management.sql

```diff
@@ -372,6 +372,13 @@ SELECT master_add_secondary_node('localhost', 9994, primaryname => 'localhost',
 SELECT master_add_secondary_node('localhost', 9993, 'localhost', 2000);
 SELECT master_add_secondary_node('localhost', 9992, 'localhost', :worker_1_port, nodecluster => 'second-cluster');
 
+SELECT nodeid AS worker_1_node FROM pg_dist_node WHERE nodeport=9992 \gset
+
+-- citus_update_node allows updating a node from the non-default cluster
+SELECT citus_update_node(:worker_1_node, 'localhost', 9991);
```

Comment:
Would be nice to have tests for `citus_nodeport_for_nodeid ` and `citus_nodename_for_nodeid ` at this point with `:nodeid`?